ahoy.js

```diff
@@ -345,27 +368,39 @@
   };
 
   ahoy.trackClicks = function () {
-    $(document).on("click", "a, button, input[type=submit]", function (e) {
-      var $target = $(e.currentTarget);
-      var properties = eventProperties(e);
-      properties.text = properties.tag == "input" ? $target.val() : $.trim($target.text().replace(/[\s\r\n]+/g, " "));
-      properties.href = $target.attr("href");
-      ahoy.track("$click", properties);
-    });
+    var elements = document.querySelectorAll("a, button, input[type=submit]");
+
+    for (var i = 0; i < elements.length; i++) {
+      elements[i].addEventListener('click', function (e) {
```

Comment:
Fixed in https://github.com/ankane/ahoy.js/pull/10/commits/c43d34dfecdb75d171d953a66922cf5be460499e but not entirely sure if this is what you meant. AFAIK this is the best way to [loop over a NodeList](https://developer.mozilla.org/nl/docs/Web/API/NodeList#Example) and it is not possible to [addEventListener on a NodeList](http://stackoverflow.com/questions/12362256/addeventlistener-on-nodelist). Just let me know what you think!